### PR TITLE
Removed BLOSSOM

### PR DESCRIPTION
**Pull request by thepagan**
still needs to have documentation and some comments updated.

Please ensure this checklist is followed for any pull requests for this repo. This checklist must be checked by both the PR creator and by anyone who reviews the PR.
* [ ] Relevant documentation for this PR has to be completed and reviewed by @mdr0id before the PR can be merged
* [ ] A test plan for the PR must be documented in the PR notes and included in the test plan for the next regular release

As a note, all buildbot tests need to be passing and all appropriate code reviews need to be done before this PR can be merged

### CODE REVIEW

File: src/gtest/test_checktransaction.cpp [hunk 1]
```diff
@@ -1002,19 +1000,19 @@ TEST(ChecktransactionTests, OverwinteredContextualCreateTx) {
         1, false, 0, 0);
     // Overwinter activates
     ContextualCreateTxCheck(params, overwinterActivationHeight,
-        OVERWINTER_TX_VERSION, true, OVERWINTER_VERSION_GROUP_ID, overwinterActivationHeight + DEFAULT_PRE_BLOSSOM_TX_EXPIRY_DELTA);
+        OVERWINTER_TX_VERSION, true, OVERWINTER_VERSION_GROUP_ID, overwinterActivationHeight + DEFAULT_PRE_EXPIRY_DELTA);
```
Comment (miodragpop) [inline src/gtest/test_checktransaction.cpp]: I would rather name that const DEFAULT_EXPIRY_DELTA than DEFAULT_PRE_EXPIRY_DELTA

File: src/main.cpp [hunk 2]
```diff
@@ -7780,10 +7780,6 @@ CMutableTransaction CreateNewContextualCMutableTransaction(const Consensus::Para
     mtx.nVersion        = txVersionInfo.nVersion;
 
     if (mtx.fOverwintered) {
-        bool blossomActive = consensusParams.NetworkUpgradeActive(nHeight, Consensus::UPGRADE_BLOSSOM);
-        unsigned int defaultExpiryDelta = blossomActive ? DEFAULT_POST_BLOSSOM_TX_EXPIRY_DELTA : DEFAULT_PRE_BLOSSOM_TX_EXPIRY_DELTA;
-        mtx.nExpiryHeight = nHeight + (expiryDeltaArg ? expiryDeltaArg.value() : defaultExpiryDelta);
```
Comment (miodragpop) [inline src/main.cpp:7785]: this line of code should be undeleted

File: src/main.h [hunk 3]
```diff
@@ -76,8 +76,7 @@ static const CAmount HIGH_MAX_TX_FEE = 100 * HIGH_TX_FEE_PER_KB;
 /** Default for -maxorphantx, maximum number of orphan transactions kept in memory */
 static const unsigned int DEFAULT_MAX_ORPHAN_TRANSACTIONS = 100;
 /** Default for -txexpirydelta, in number of blocks */
-static const unsigned int DEFAULT_PRE_BLOSSOM_TX_EXPIRY_DELTA = 20;
-static const unsigned int DEFAULT_POST_BLOSSOM_TX_EXPIRY_DELTA = DEFAULT_PRE_BLOSSOM_TX_EXPIRY_DELTA * Consensus::BLOSSOM_POW_TARGET_SPACING_RATIO;
+static const unsigned int DEFAULT_TX_EXPIRY_DELTA = 20;
```
Comment (miodragpop) [inline src/main.h:79]: with or without TX in the name?

File: src/gtest/test_checktransaction.cpp (hunk 1 above)
Comment (thepagan) [reply src/gtest/test_checktransaction.cpp]:
```suggestion
        OVERWINTER_TX_VERSION, true, OVERWINTER_VERSION_GROUP_ID, overwinterActivationHeight + DEFAULT_TX_EXPIRY_DELTA);
```

**Pull request merged**